Gemfile

```diff
@@ -0,0 +1,5 @@
+source 'https://rubygems.org'
+
+gem 'rspec'
+
+gem 'rubocop', require: false
```

Comment:
What does `require: false` mean?

Reply:
It seems the reason why require: false is suggested is because RuboCop is designed to be run as a CLI and not to be loaded as part of the environment when the application is started.

If you don't set require: false, Rails will load the gem. Even if the gem doesn't conflict with the app process, if you don't need it then loading the library it's an unnecessary overhead (both in terms of memory and allocation).

Reply:
Ah then presumably all gems that are not designed to be run as part of the application itself should be marked with `require: false`, so I would add that to line 3 for the `rspec` gem.

Reply:
I'm following convention; I've never seen it next to the rspec gem.

Reply:
Cool, let's leave it like this for now then, and we'll probably change it soon anyway when we find the problems :)